src/Tizen.Uix.InputMethod/Tizen.Uix.InputMethod/InputMethodEditor.cs

```diff
@@ -2139,5 +2139,80 @@ public static void SetFloatingDragEnd()
                 throw InputMethodExceptionFactory.CreateException(error);
             }
         }
+
+        /// <summary>
+        /// Notify if the languate is updated.
```

Comment:
I can't understand this descriptions.

when need to call thia api?
and what is happened this api called

Reply:
This API must be called when IME's language has been changed.
When this API is called, the widget uses the C API to get language.

Reply:
How about change API name to "SendLanguageUpdated" 
and Description to "Notify that the language was updated."

Reply:
1. Who is use this API?
=> Yes. The developer of keyboard app uses this API.

2. When called this API, who is got a event? Could you let me know related API to receive this event?
=> The Widget receives language event.